### ISSUE TITLE
Button flatness

### ISSUE THREAD

**Issue by godlyranchdressing** [ubuntu/gnome-shell-communitheme#160]
@Feichtmeier mentioned that all buttons are going to be flat somewhere but I **really** don't think that this should extend to dialog buttons. The rest of the buttons being flat I can live with but these should pop out against the background.

---

The border around the notification clear all button is a little unnecessary:

![bf](https://user-images.githubusercontent.com/27529229/39582630-5a7da920-4ebc-11e8-9aef-f4dc6b100a34.png)
![af](https://user-images.githubusercontent.com/27529229/39582629-5a556d98-4ebc-11e8-97ac-efaccf2fa245.png)

---

The button pressed effect needs to be updated.

![dd](https://user-images.githubusercontent.com/27529229/39582740-9df12ce0-4ebc-11e8-9cc6-b78980b91d67.gif)

![pra](https://user-images.githubusercontent.com/27529229/39582819-c1f14af8-4ebc-11e8-8727-dd8a45b97cec.gif)

**Comment by clobrano** [comment]
> The border around the notification clear all button is a little unnecessary

I actually like this little guidance to the buttons, but in fact or we use it everywhere or nowhere

**Comment by Feichtmeier** [comment]
I think we mix up some stuff here.

1) the whole shell is flat
2) all icon buttons in popovers are flat
3) IMHO dialog buttons should be flat too, at least in unpressed state , my reference here would be the notifications

**Comment by godlyranchdressing** [comment]
@clobrano yeah, the border isn't used for any other buttons, even flat ones.

![system](https://user-images.githubusercontent.com/27529229/39584881-7a95dbe2-4ec1-11e8-9780-9f5a88859e1c.png)

@Feichtmeier 
>Look at the notifications, like that!:)

I think I get it now, but in that case the button shouldn't look like a notification. I'm just unconvinced by the current design for flat dialog buttons. We can flatten out them out by removing the old style's highlight to give us something like this:

![aft](https://user-images.githubusercontent.com/27529229/39585687-5e3abb00-4ec3-11e8-8d37-2150c3ee8caf.png)

Or leave the old style's highlight and make the button color less differential (they did become too light after the shift to a brownish-black).
![w](https://user-images.githubusercontent.com/27529229/39585156-1d8a48c4-4ec2-11e8-898f-d4aa89eebcef.png)

**Comment by madsrh** [comment]
> I actually like this little guidance to the buttons, but in fact or we use it everywhere or nowhere

I agree with @clobrano. I know that the shell can and will be different than the GTK theme, but more visual guidance to borderless buttons (the cancel button in the [link](https://user-images.githubusercontent.com/30781876/36177045-c6b1709e-1114-11e8-8ac0-a91f07452768.png)) was also requested [here](https://github.com/ubuntu/gtk-communitheme/issues/152).

**Comment by Feichtmeier** [comment]
@godlyranchdressing please flat out the cancel button in GDM/Lockscreen too. I've closed my old issue about this.

**Comment by godlyranchdressing** [comment]
How about this?

$button_bg_color: lighten($bg_color, 5%):

![aaa](https://user-images.githubusercontent.com/27529229/39587399-65be8fb0-4ec7-11e8-88c0-00689f161b2b.gif)

vs $button_bg_color: lighten($bg_color, 8%):

![4t](https://user-images.githubusercontent.com/27529229/39587487-9dc9c3e8-4ec7-11e8-91c8-325aff5c4f87.gif)

**Comment by Feichtmeier** [comment]
2. works well! I was just fearing that you want to bring back the 3d buttons :sob: 
Could you apply this also to the cancel button in GDM/lock screen?

**Comment by clobrano** [comment]
Black border works fine, but I like the current bright one as well, which incidentally might get along well with the last iteration of https://github.com/ubuntu/gtk-communitheme/pull/394

**Comment by godlyranchdressing** [comment]
I miss the highlight. :disappointed:
